N24DCCN058/Baitapbuoi5/Bai2.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+using namespace std;
+int main() {
+    int n  ; 
+    cin >> n ; 
+    while ( n > 10){
+        cin >> n ;
+    }
+    for (int i = 1 ; i < 11 ; i++){
+        cout << i*n << endl ; 
```

Comment:
In ra bảng cửu chương, k phải in ra kết quả.

Reply:
là sao anh :V

Reply:
phải in như 1x2 = 2. còn em chỉ in 2 thôi, k in dc phép tính